annotations.md

```diff
@@ -15,7 +15,7 @@ Consumers MUST NOT generate an error if they encounter an unknown annotation key
 
 ## Pre-Defined Annotation Keys
 
-This specification defines the following annotation keys, intended for but not limited to manifest list and image manifest authors:
+This specification defines the following annotation keys, intended for but not limited to [image-index](image-index.md) and image [manifest](manifest.md) authors:
```

Comment:
Hyphenated "image-index" here (you seem to prefer the unhyphenated "image index").